src/gl/vcFramebuffer.h

```diff
@@ -16,7 +16,7 @@ enum vcFramebufferClearOperation
 
 struct vcFramebuffer;
 
-bool vcFramebuffer_Create(vcFramebuffer **ppFramebuffer, vcTexture *pTexture, vcTexture *pDepth = nullptr, uint32_t level = 0);
+bool vcFramebuffer_Create(vcFramebuffer **ppFramebuffer, vcTexture *pTexture, vcTexture *pDepth = nullptr, uint32_t level = 0, vcTexture *pAttachment2 = nullptr);
```

Comment:
Should we perhaps consider creating a `vcFramebuffer_CreateAdv` which takes an array of attachments which indicate what kind of attachment they are and the `vcTexture*` to use?
```c++
enum vcFramebufferAttachmentType
{
  vcFAT_Colour,
  vcFAT_Depth
};

struct vcFramebufferAttachment
{
  vcTexture *pTexture;
  vcFramebufferAttachmentType type;
};
```

Reply:
Yeah sounds good - this change would be consistent with the vcTexture_CreateAdv() API. Probably won't make this change for this PR though.

Reply:
> Probably won't make this change for this PR though.

Minor issue with going down this path is that `vcFramebuffer_Create` will need to have this change undone, and all usages moved to use `vcFramebuffer_CreateAdv`.